### PR TITLE
[4.0]namespace administrator components - com_menus

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_menus/layouts/joomla/searchtools/default.php [hunk 1]
```diff
@@ -83,15 +87,15 @@
 		}
 		?>
 		<div class="js-stools-container-bar">
-			<?php echo JLayoutHelper::render('joomla.searchtools.default.bar', $data); ?>
+			<?php echo LayoutHelper::render('joomla.searchtools.default.bar', $data); ?>
```
Comment (Quy) [inline administrator/components/com_menus/layouts/joomla/searchtools/default.php:91]: Add `use Joomla\CMS\Layout\LayoutHelper;`

File: administrator/components/com_menus/tmpl/items/default_batch_body.php [hunk 2]
```diff
@@ -8,15 +8,18 @@
  */
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/items/default_batch_body.php]: Remove duplicate

File: administrator/components/com_menus/tmpl/items/modal.php [hunk 3]
```diff
@@ -9,67 +9,74 @@
 
 defined('_JEXEC') or die;
 
+use Joomla\CMS\HTML\HTMLHelper;
```
Comment (Quy) [inline administrator/components/com_menus/tmpl/items/modal.php]: Remove duplicate

File: administrator/components/com_menus/layouts/joomla/searchtools/default.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_menus/layouts/joomla/searchtools/default.php:91]: done

File: administrator/components/com_menus/tmpl/items/default_batch_body.php (hunk 2 above)
Comment (brianteeman) [reply administrator/components/com_menus/tmpl/items/default_batch_body.php]: done

File: administrator/components/com_menus/tmpl/items/modal.php (hunk 3 above)
Comment (brianteeman) [reply administrator/components/com_menus/tmpl/items/modal.php]: done

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**